acceptance/acceptance_test.go

```diff
@@ -659,7 +656,11 @@ func runTest(t *testing.T,
 
 	// Include exit code in output (if non-zero)
 	formatOutput(out, err)
-	require.NoError(t, out.Close())
+
+	// Write the output from the script execution to output.txt.
+	outputPath := filepath.Join(tmpDir, "output.txt")
+	err = os.WriteFile(outputPath, out.Bytes(), 0o644)
```

Comment:
One difference I see is that WriteFile will close the file right away https://cs.opensource.google/go/go/+/refs/tags/go1.25.0:src/os/file.go;l=941

whereas previously we relied on defer out.Close()


what if we do that - explicitly close the file as soon as possible?